plugin/source/secretsmanager/secretsmanager.go

```diff
@@ -0,0 +1,134 @@
+package secretsmanager
+
+/*
+	Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+*/
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+
+	"github.com/facebookincubator/go2chef"
+	"github.com/mitchellh/mapstructure"
+)
+
+// TypeName is the name of this source plugin
+const TypeName = "go2chef.source.secretsmanager"
+
+// Source implements an AWS secretsmananger source plugin that
+// copies data stored in AWS secrets mananger into a file.
+type Source struct {
+	logger go2chef.Logger
+
+	SourceName  string `mapstructure:"name"`
+	Region      string `mapstructure:"region"`
+	SecretId    string `mapstructure:"secret_id"`
+	FileName    string `mapstructure:"filename"`
+	Credentials struct {
+		AccessKeyID     string `mapstructure:"access_key_id"`
+		SecretAccessKey string `mapstructure:"secret_access_key"`
+		Token           string `mapstructure:"token"`
+	}
+}
+
+func (s *Source) String() string {
+	return "<" + TypeName + ":" + s.SourceName + ">"
+}
+
+// Name returns the name of this source instance
+func (s *Source) Name() string {
+	return s.SourceName
+}
+
+// Type returns the type of this source
+func (s *Source) Type() string {
+	return TypeName
+}
+
+// SetName sets the name of this source instance
+func (s *Source) SetName(name string) {
+	s.SourceName = name
+}
+
+// DownloadToPath performs reads the SecretString from secretsmanager and
```

Comment:
You can drop this word and it reads better.
```suggestion
// DownloadToPath reads the SecretString from secretsmanager and
```